src/index.html

```diff
@@ -11,7 +11,111 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="http://localhost:8080/api">
+      <fieldset class="group">
+        <legend>Personal information</legend>
+        <label class="form-filed">
+          Surname:
+          <input type="text" autocomplete="off"
+          name="surname" required
+          >
```

Comment:
[checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md?rgh-link-date=2022-09-27T08%3A01%3A14Z) 3rd item, format your attributes correctly EVERYWHERE where you have more than 2 attributes
```suggestion
          <input 
            type="text" 
            autocomplete="off"
            name="surname" 
            required
          >
```